Automattic/wp-calypso
# Emails: Improve UX for Google Workspace account & mail boxes creation

#### Changes proposed in this Pull Request

This change fixes break-points in Google Workspace comparison page (it is a bit weird plus differs from Professional Email breakpoints), adds a small margin (64px) in the right features pane, and replaces "remove this user" with "remove this mailbox" wording.

#### Testing instructions

1. Go to upgrades -> Emails
2. Visual inspection of the Google Workspace section to add an account/mailbox as per the below gifs

BEFORE | AFTER
------------ | -------------
![non-fixed-breaks](https://user-images.githubusercontent.com/5689927/132539347-bf6a7940-31ef-491b-a143-7369b05600ab.gif) | ![upsell-workspace-google](https://user-images.githubusercontent.com/5689927/133931116-01d7ca57-7c2f-402f-a3ad-becf01588d1f.gif)

It was also needed to update the main header for the comparison page, check its behavior:

BEFORE | AFTER
------------ | -------------
![header-old](https://user-images.githubusercontent.com/5689927/132996170-b52cf4a6-8291-42eb-9835-183e87876967.gif) | ![header-comparison](https://user-images.githubusercontent.com/5689927/133931120-2bbf9b73-8a82-4219-b570-5b2531515a00.gif)

#### Remarks

As part of this change was also needed to update the three icon sizes/behaviors, the Forwarding, Professional Email, and Google Workspace. 

Add mailboxes has been checked too as it might have regression bugs.
![add-mailbox-google](https://user-images.githubusercontent.com/5689927/133931229-cd7413fb-8ed4-47b3-ba56-9867f0976b2c.gif)

Related to {1200182182542585-as-1200879615791996}
